viam-cartographer_test.go

```diff
@@ -215,16 +215,14 @@ func TestEndpointFailures(t *testing.T) {
 	test.That(t, frame, test.ShouldBeEmpty)
 	test.That(t, fmt.Sprint(err), test.ShouldContainSubstring, "error getting SLAM position")
 
-	// This test needs to be updated after the slam service interface includes GetPointCloudMap https://viam.atlassian.net/browse/RSDK-2246
-	callbackPointCloud, err := svc.GetPointCloudMapStream(context.Background(), "hi")
+	callbackPointCloud, err := svc.GetPointCloudMap(context.Background(), "hi")
```

Comment:
[nit] Can you change `"hi"` to something more descriptive of what this string represents? 

Additionally, can you add to the function description of `GetPointCloudMap` what `name` is supposed to be? I tried to look up what `"hi"` stands for, and there is no documentation for it. I know this is out of scope for this ticket, but it would help make this test and the function definition easier to understand.